cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java

```diff
@@ -214,9 +221,79 @@ public void init() {
 	
     public Cocos2dxGLSurfaceView onCreateView() {
     	Cocos2dxGLSurfaceView glSurfaceView = new Cocos2dxGLSurfaceView(this);
-
-    	glSurfaceView.setEGLConfigChooser(this.glContextAttrs[0], this.glContextAttrs[1],this.glContextAttrs[2],
-    		this.glContextAttrs[3],this.glContextAttrs[4],this.glContextAttrs[5]);
+    	//this line is need on some device if we specify an alpha bits
+    	if(this.glContextAttrs[3] > 0) glSurfaceView.getHolder().setFormat(PixelFormat.TRANSLUCENT);
```

Comment:
What kinds of problem if don't assign `PixelFormat.TRANSLUCENT` to GLSurfaceView?

Reply:
not a reasonable adaptation, `need on some device` but case transparent issues on some other device.

refre to the discuss in forum: https://forum.cocos.org/t/cocos/81491

Reply:
Yep, i don't understand why need this line too. I think it is time to remove it.